src/components/button/button.tsx

```diff
@@ -358,4 +369,9 @@ export class Button
       resetForm(this);
     }
   };
+
+  private handleSlotchange = (event): void => {
+    const textNodes = event.target.assignedNodes().filter((node) => node.nodeName === "#text");
+    this.initialSlottedText = textNodes.map((node) => node.textContent).join("");
```

Comment:
Is there a reason you need to access the text nodes? Seems like a container around the slot should suffice for getting the offsetWidth/scrollWidth.

Reply:
Yeah, you're right. Not sure why I couldn't get it to work before. I think I needed to clean up the demo page noise to see the output more clearly.